solidity/contracts/OngoingAirdrops.sol

```diff
@@ -111,7 +111,49 @@ contract OngoingAirdrops is AccessControl, IOngoingAirdrops {
     TokenAmount[] calldata _tokensAmounts,
     address _recipient,
     bytes32[] calldata _proof
-  ) internal virtual {}
+  ) internal virtual {
+    // Basic checks
+    if (_campaign == bytes32(0)) revert InvalidCampaign();
+    if (_claimee == address(0) || _recipient == address(0)) revert ZeroAddress();
```

Comment:
Is the `_claimee == address(0)` check necessary? It won't be on the merkle tree, so it should fail later, right?